src/hist.jl

```diff
@@ -82,6 +82,7 @@ Plots.group_as_matrix(g::GroupedHist) = true
     edges = h.edges[1]
     bar_width --> mean(map(i -> edges[i+1] - edges[i], 1:nbins))
     x = map(i -> (edges[i] + edges[i+1])/2, 1:nbins)
+    ntot = count(x->!isnan(x), v) 
```

Comment:
Since this is only needed if `group !== nothing`, can you moved it to below the `else`?